networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc

```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="albo-sts-cluster"]
-= Preparing for the AWS Load Balancer Operator on a cluster using the AWS {sts-full}
+= Preparing for the AWS Load Balancer Operator on a cluster using the AWS
```

Comment:
You can leave the  {sts-full} in the title. Just move the common-attributes. See [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata).

```
:_mod-docs-content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="albo-sts-cluster"]
= Preparing for the AWS Load Balancer Operator on a cluster using the AWS {sts-full}
:context: albo-sts-cluster
```

Reply:
> Version(s): 4.14 and 4.15

I assume, also, this should me CP'd to 4.16 also?